### ISSUE TITLE
ShakeShake model does not learn with default hyperparams

### ISSUE THREAD

**Issue by nottombrown** [tensorflow/tensor2tensor#373]
After running for 15K steps with the default hyperparams, shake shake isn't classifying better than chance.

![image](https://user-images.githubusercontent.com/306655/31921549-d4f423fc-b824-11e7-8d8d-6c5a05646b4e.png)

From an email from @rshin :
> I've gotten the model to train reasonably when using shakeshake_type=equal,learning_rate=1e-2 (after 10k steps, eval accuracy is ~70%; I haven't run it longer than that yet).

I'm currently running an experiment to see if it converges to a reasonable score when running with those hyperparameters.

Either way, I would vote to put a warning in the docstring for this model so that people don't run it and expect it to perform better than it does.

**Comment by rsepassi** [comment]
Adding a warning in the next release. Thanks!
(And do let us know if you find good hparam settings)
